models/DataObject/ClassDefinition/Data/FieldDefinitionEnrichmentInterface.php

```diff
@@ -27,7 +27,7 @@ interface FieldDefinitionEnrichmentInterface
      *
      * @param array $context
      *
-     * @return Data
+     * @return $this
      */
-    public function enrichFieldDefinition(/** array */ $context = []) /** : Data */;
+    public function enrichFieldDefinition(/** array */ $context = []) /** : static */;
```

Comment:
```suggestion
    public function enrichFieldDefinition(/** array */ $context = []) /** : self */;
```

Reply:
Returns always `$this`, so it should be static return type